pkg/cloud/meta/service.go

```diff
@@ -116,6 +117,19 @@ func (i *ServiceInfo) ListItemName() string {
 	return "Items"
 }
 
+func (i *ServiceInfo) NetworkServicesFmt() string {
```

Comment:
Why don't you use the Stringer https://go.dev/tour/methods/17 implementing the `String()`method for this?

Reply:
Don't completely follow your comment, can you explain more?